app/routes/helper_methods.py

```diff
@@ -0,0 +1,38 @@
+from flask import abort,make_response
+from sqlalchemy import asc, desc
+from ..db import db
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except ValueError:
+        response = {"message": f"{cls.__name__} id {model_id} is invalid"}
+        abort(make_response(response, 400))
+
+    query = db.select(cls).where(cls.id == model_id)
+    model = db.session.scalar(query)
+
+    if not model:
+        response = {"message": f"{cls.__name__} with id {model_id} not found"}
+        abort(make_response(response, 404))
+
+    return model
+
+def helper_model_from_dict(cls, request_body, required_fields = ["title", "description"]):
+    for field in required_fields:
```

Comment:
I feel like `required_fields` having default values isn't the best choice for this function. What if we want to create a hypothetical model `User` that doesn't have `"title"` or `"description"` but has "name" and "address".

If this function is invoked and `required_fields` is accidentally omitted from the call, then the function would incorrectly use `"title"` or `"description"`. I'd prefer the function to throw a TyprError and specify something like "missing 1 required positional argument: 'required_fields'" instead of applying incorrect default values.

```suggestion
def helper_model_from_dict(cls, request_body, required_fields):
    for field in required_fields:
```